source/go-perun/index.rst

```diff
@@ -0,0 +1,49 @@
+Go-Perun
```

Comment:
There are now multiple top-level sites:  
`Introduction`, `Concepts`, `Go-Perun` and `Perun-node`.  
I think this is a bit confusion to someone who opens the page for the first time. We should at least rename `Go-Perun` to `Tutorial`.

Reply:
I think this has to do how sphinx documentation pages work. The idea is that go-perun can cover anything related to the go-perun Framework component, not just the tutorial.